Gemfile

```diff
@@ -53,6 +53,9 @@ gem "bootsnap", require: false
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
   gem "debug", platforms: %i[ mri mingw x64_mingw ]
+  gem "rspec-rails"
+  gem "factory_bot_rails"
+  gem 'shoulda-matchers'
```

Comment:
rearrange the gemfile in alphabetical order

Reply:
rearranged. Thanks